### PR TITLE
spread, tests: Fedora 32 is EOL, drop it

### PR DESCRIPTION
**Pull request by bboozzoo**
Fedora 32 reached EOL on 25.05.2021. We can drop it from the testing suite now.

### CODE REVIEW

File: tests/main/snap-routine-portal-info/task.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@ summary: The portal-info command provides information about a confined process
 
 systems:
     - -ubuntu-14.04-*
-    - -fedora-32-*  # TODO: add support for cgroupv2 based tracking
     - -fedora-33-*
```
Comment (bboozzoo) [inline tests/main/snap-routine-portal-info/task.yaml:5]: Bit surprised we didn't have to add Fedora 34 here.

**Review by stolowski — APPROVED** [review]
Thanks

**Review by sergiocazzolato — COMMENTED** [review]
Should we remove the packaging for fedora-32, right?

**Review by sergiocazzolato — APPROVED** [review]
Nice, thanks for the change

**Review by anonymouse64 — APPROVED** [review]
lgtm, thanks for cleaning up the old fedora releases in packaging/ too

**Pull request merged**